testing/cypress/e2e/tests/user-content-access.cy.js

```diff
@@ -0,0 +1,154 @@
+/// <reference types="cypress" />
+
+describe('Test User Role Access to Content Moderation', () => {
+
+  it('Test Site Admin', () => {
+    cy.signin(Cypress.env('roles').site_admin.username, Cypress.env('test_pass'))
+
+    // Can create a draft
+    cy.visit('/node/add/page')
+    cy.get('[data-drupal-selector="edit-title-0-value"]').type('Test Page')
+
+    cy.get('[class="ck-blurred ck ck-content ck-editor__editable ck-rounded-corners ck-editor__editable_inline"]').then(field => {
+      cy.get(field[0]).clear().realType('test page')
+      cy.get(field[1]).clear().realType('test page')
+
+    })
+    cy.get('[class="claro-details__summary claro-details__summary--accordion-item"]').then(dropdown => {
+      cy.get(dropdown[3]).click()
+      cy.get('[data-drupal-selector="edit-path-0-pathauto"]').click()
+      cy.get('[data-drupal-selector="edit-path-0-alias"]').type('/cypress-content-moderation-test')
+    })
+    cy.get('[data-drupal-selector="edit-submit"]').click()
+    cy.url().should('contain', 'cypress-content-moderation-test')
+
+    // Can view draft content 
+    cy.request('/cypress-content-moderation-test').then((response) => {
+      expect(response.status).to.eq(200)})
+   
+    // Can modify draft content
+    cy.visit('/cypress-content-moderation-test')
+    cy.get('[class="usa-button-group__item"]').then(btn => {
+      cy.get(btn[1]).click()
+    })
+    cy.url().should('contain', 'edit')
```

Comment:
i found when you make a new page the edit URL will change bases on the number so it would be `node/93/edit` and the when I reran that test it would be `node/94/edit` so switched this up to hit the edit button and very that the URL contains edit